### DISCUSSION TITLE
Modeling a photobioreactor

### DISCUSSION THREAD

**Discussion by arobison** [BioSTEAMDevelopmentGroup/biosteam#166] [Q&A]
Hi all!

Are there any public examples of using BioSTEAM to model a photobioreactor (PBR)? I'd like to build a simulation of using a PBR for algae cultivation and could use some pointers on getting started. Thanks!

~Austin

**Comment by yoelcortes** [comment]
@Ga-YeongKim, @joyxyz1994, @yalinli2, @sarangbhagwat,

I know a few of you were working on a project with algae. Whenever you get a chance, please direct @arobison to any models that are ready for public use (in BioSTEAM or QSDsan; if any).

Thanks!

**Reply by yalinli2** [reply]
haha I literally just slacked Ga-Yeong a min ago, will see how she replies

**Reply by arobison** [reply]
Thanks all! Very curious to dig in here.

**Reply by GaYeongKim** [reply]
Hi @arobison ! We have an algae model under development and hope to submit it in the next 2-3 months. When the paper is published, the source code will also be made available to the public. If you would like to potentially use it sooner, we would be glad to chat with you to learn more about your application (email/zoom, etc.). Thank you!

**Reply by arobison** [reply]
Thanks Ga-Yeong! I'd love to learn more and happy to be a beta tester. I'll drop you an email and we can set up some time to chat.
